crypto/asn1/charmap.pl

```diff
@@ -82,8 +82,8 @@
 
 # Now generate the C code
 
-my @lt = localtime;
-my $YEAR = $lt[5] + 1900;
+# Output year depends on the year of the script.
+my $YEAR = [localtime([stat($0)]->[9])]->[5] + 1900;
```

Comment:
Uhmmmmm, no.  The *output* file might change even if this script file doesn't, and it's the *output* file that we need to update the year on, based on how *it* changes compared to previously.